drivers/hap_report/app/views/hap_report/warehouse_reports/hap_reports/_filter.haml

```diff
@@ -7,7 +7,9 @@
         = f.input :end, as: :date_picker
       .col-sm-6
         = f.input :project_ids, as: :grouped_select_two, collection: HapReport::Report.hap_projects(current_user), group_method: :last, input_html: { multiple: true }, label: 'Projects'
-
+    .row
+      .col
+        = f.input :erap_csv, as: :file, label: 'Optional eRAP Data CSV'
```

Comment:
I feel like we need something here that details the file shape, maybe in a collapsible?